scripts/test/support.py

```diff
@@ -157,7 +158,9 @@ def run_command(cmd, expected_status=0, stderr=None, expected_err=None):
   out, err = proc.communicate()
   if proc.returncode != expected_status:
     raise Exception(('run_command failed', err))
-  if expected_err is not None and err != expected_err:
+  err_correct = expected_err is None or \
```

Comment:
The full stderr of mozjs for successful test looks like `a.2asm.js:warning: Successfully compiled asm.js code (total compilation time 4ms; caching disabled by missing command-line arguments)`. The only part the test cares about is `Successfully compiled asm.js code`. I don't know of a way to stop the timing information from being emitted, which would allow us to use equality here.